examples/multiomics_mogonet/README.md

```diff
@@ -0,0 +1,60 @@
+# Multiomics Data Integration using Graph Convolutional Networks (MOGONET)
+
+## 1. Description
+
+Multimodal learning for multiomics data analysis is a promising research area in biomedical studies. The integration of multiple modalities (e.g., genomics, epigenomics, transcriptomics, proteomics, metabolomics , etc.) by capturing their complementary information provides a deeper understanding of most complex human diseases. This example is constructed by refactoring the code of [MOGONET](https://doi.org/10.1038/s41467-021-23774-w) [1], a multiomics integrative method for cancer classification tasks, using the PyTorch Geometric and PyTorch Lightning frameworks.
+
+
+## 2. MOGONET
+
+**M**ulti-**O**mics **G**raph c**O**nvolutional **NET**works ([MOGONET](https://doi.org/10.1038/s41467-021-23774-w)) is
+a multiomics fusion framework for cancer classification and biomarker identification that utilizes supervised graph
+convolutional networks for omics datasets. The overall framework of MOGONET is illustrated below.
+
+![MOGONET Architecture](image/MOGONET.png)
+
+The original implementation of MOGONET is available [here](https://github.com/txWang/MOGONET.git).
```

Comment:
The link should be https://github.com/txWang/MOGONET

Reply:
The link has been updated.